### PR TITLE
Integrate 202208 based changes

### PR DESCRIPTION
**Pull request by kuqin12**
This PR integrates the override changes checked into upstream EDK2 between 202202 and 202208 stable tags, with developer's discretion. The branch is verified bootable on QEMU Q35 platform.

A full list of changes involved from EDK2:

https://github.com/tianocore/edk2/commit/2aa107c0aa2e1375651867c8df1b81ff64b67fce
https://github.com/tianocore/edk2/commit/4a68176cb548902cad93e6ebb5957d0cac94d297
https://github.com/tianocore/edk2/commit/86d41c077eaa4b78f71b012671fdd8a7549fc7a2
https://github.com/tianocore/edk2/commit/83d5871184d1e09332565bfc939e5fc8354b5b79
https://github.com/tianocore/edk2/commit/7b4754904efd5503d191f034ef17e982ceb65962
https://github.com/tianocore/edk2/commit/d3febfd9ade35dc552df6b3607c2b15d26b82867
https://github.com/tianocore/edk2/commit/818bc9596d720b43c369e57be4ad1fd8f7e1917b
https://github.com/tianocore/edk2/commit/92288f433485e84863047fae698614c6785869d1
https://github.com/tianocore/edk2/commit/ab9d790901c13b458fab3637efdf53a425bc7502
https://github.com/tianocore/edk2/commit/b6b2de8848648bbba1ab74b0752d6650bc03018a
https://github.com/tianocore/edk2/commit/d74e93268185273b2a91ff8577659bafeac4341c
https://github.com/tianocore/edk2/commit/3571fc906f7462556881f6ee153565b3595cee10

### CODE REVIEW

File: basecore_ext_dep.yaml [hunk 1]
```diff
@@ -9,6 +9,6 @@
   "name": "MU_BASECORE",
   "var_name": "BASECORE_PATH",
   "source": "https://github.com/microsoft/mu_basecore.git",
-  "version": "661901f38f33606cc6cdbce7defccfc8b80a84af", # release/202202
+  "version": "0d04a5e280c55eb742deaabcfa97da85a0afe7b2", # release/202208
```
Comment (kenlautner) [inline basecore_ext_dep.yaml:12]: Basecore has more recent commits than this one.

File: mu_plus_ext_dep.yaml [hunk 2]
```diff
@@ -9,6 +9,6 @@
   "name": "MU_PLUS",
   "var_name": "MU_PLUS_PATH",
   "source": "https://github.com/microsoft/mu_plus.git",
-  "version": "379bd43e71edc1de79710a6bb13ad0db20cfdd59", # release/202202
+  "version": "a298f6a77d74cf1e293f9e985eaf8ab7b59789c7", # release/202208
```
Comment (kenlautner) [inline mu_plus_ext_dep.yaml:12]: MU_PLUS has new commits as well.

**Review by kenlautner — APPROVED** [review]

File: mu_plus_ext_dep.yaml (hunk 2 above)
Comment (kuqin12) [reply mu_plus_ext_dep.yaml]: Thanks, updated.

File: basecore_ext_dep.yaml (hunk 1 above)
Comment (kuqin12) [reply basecore_ext_dep.yaml]: Thanks, updated.

**Pull request merged**